create-projects/create-projects.sh

```diff
@@ -1,88 +1,117 @@
+oc policy add-role-to-group system:image-puller system:serviceaccounts:${PROJECT_ID}-dev -n "${PROJECT_ID}-cd"
+
+echo "Grant serviceaccount 'default' role 'image-builder' to import images from other cluster"
+oc policy add-role-to-user system:image-builder --serviceaccount default -n "${PROJECT_ID}-dev"
+oc policy add-role-to-user system:image-builder --serviceaccount default -n "${PROJECT_ID}-test"
+
+echo "Seed admins, by default only role 'dedicated-admin' has admin rights"
+if [ -n ${PROJECT_ADMIN} ]; then
+  for admin_user in $(echo $PROJECT_ADMIN | sed -e 's/,/ /g'); do
+    echo "- seeding admin: ${admin_user}"
+    oc policy add-role-to-user admin ${admin_user} -n "${PROJECT_ID}-dev"
+    oc policy add-role-to-user admin ${admin_user} -n "${PROJECT_ID}-test"
+    oc policy add-role-to-user admin ${admin_user} -n "${PROJECT_ID}-cd"
+  done
 fi
 
+if [ -n ${PROJECT_GROUPS} ]; then
+  echo "Seeding special permission groups"
+  for group in $(echo $PROJECT_GROUPS | sed -e 's/,/ /g'); do
+    groupName=$(echo $group | cut -d "=" -f1)
+    groupValue=$(echo $group | cut -d "=" -f2)
+
+    usergroup_role=edit
+    admingroup_role=admin
+    readonlygroup_role=view
+
+    if [ ${groupValue} == "" ]; then
+      continue
+    fi
+
+    echo "- seeding groups: ${groupName^^} - ${groupValue}"
+    if [[ ${groupName^^} == *USERGROUP* ]]; then
+      oc policy add-role-to-group ${usergroup_role} ${groupValue} -n "${PROJECT_ID}-dev"
+      oc policy add-role-to-group ${usergroup_role} ${groupValue} -n "${PROJECT_ID}-test"
+      oc policy add-role-to-group ${usergroup_role} ${groupValue} -n "${PROJECT_ID}-cd"
+    elif [[ ${groupName^^} == *ADMINGROUP* ]]; then
+      oc policy add-role-to-group ${admingroup_role} ${groupValue} -n "${PROJECT_ID}-dev"
+      oc policy add-role-to-group ${admingroup_role} ${groupValue} -n "${PROJECT_ID}-test"
+      oc policy add-role-to-group ${admingroup_role} ${groupValue} -n "${PROJECT_ID}-cd"
+    elif [[ ${groupName^^} == *READONLYGROUP* ]]; then
+      oc policy add-role-to-group ${readonlygroup_role} ${groupValue} -n "${PROJECT_ID}-dev"
+      oc policy add-role-to-group ${readonlygroup_role} ${groupValue} -n "${PROJECT_ID}-test"
+      oc policy add-role-to-group ${readonlygroup_role} ${groupValue} -n "${PROJECT_ID}-cd"
+    fi
+  done
+else
+  echo "Allow all authenticated users to view the project"
+  oc policy add-role-to-group view system:authenticated -n "${PROJECT_ID}-dev"
+  oc policy add-role-to-group view system:authenticated -n "${PROJECT_ID}-test"
+  oc policy add-role-to-group view system:authenticated -n "${PROJECT_ID}-cd"
+
+  echo "Allow all authenticated users to edit the project"
+  oc policy add-role-to-group edit system:authenticated -n "${PROJECT_ID}-dev"
+  oc policy add-role-to-group edit system:authenticated -n "${PROJECT_ID}-test"
+  oc policy add-role-to-group edit system:authenticated -n "${PROJECT_ID}-cd"
+fi
```

Comment:
The diff looks huge but the changes are very few - I think this is mostly whitespace :(